data/json/items/ranged/archery.json

```diff
@@ -195,6 +199,7 @@
     "color": "green",
     "description": "A fletched carbon fiber arrow shaft with an expanding broadhead tip.",
     "material": [ { "type": "kevlar_rigid", "portion": 81 }, { "type": "steel", "portion": 19 } ],
+    "delete": { "flags": [ "NONCONDUCTIVE" ] },
```

Comment:
Google says carbon fiber is non conductive? Google can easily be wrong but figured I'd ask.

Reply:
Happy to change it if so!  My first round of googling turned up otherwise, but I can do a deeper dive tomorrow and find sources.

Reply:
Here’s my first quick source in case it’s sufficient:

https://www.sparteccomposites.com/2022/12/22/is-carbon-fiber-conductive-a-closer-look-at-its-electrical-conductivity/

“At Spartec Composites, we are experienced carbon fiber composite manufacturers… Some carbon fiber products can be a good conductor for electricity, while others are less than ideal. However, as a general statement, carbon fiber is conductive.“

Apparently pure carbon fiber is *not* conductive on its own, but it’s also apparently really common to composite it with conductive metals, including sometimes with arrows.